pkg/acquisition/modules/file/file.go

```diff
@@ -0,0 +1,358 @@
+package fileacquisition
```

Comment:
maybe `file` is enough and more readable?

Reply:
we decided to keep the long explicit names, but alternative is welcomed